docs/user_guide/response_cache.md

```diff
@@ -28,6 +28,8 @@
 
 # Triton Response Cache
 
+**This feature is currently in beta and may be subject to change.**
```

Comment:
This may not be true. where are you getting this information?

Reply:
Looks like I missed the PR for beta tag update to the Response cache. Will merge it in